packages/build-utils/src/monorepos/monorepo-managers.ts

```diff
@@ -0,0 +1,53 @@
+import type { Framework } from '@vercel/frameworks';
+
+/**
+ * The supported list of monorepo managers.
+ *
+ * This list is designed to work with the @see {@link detectFramework} function.
+ *
+ * @example
+ *   import { monorepoManagers as frameworkList } from '@vercel/build-utils'
+ *   import { detectFramework } from '@vercel/build-utils'
+ *
+ *   const fs = new GitDetectorFilesystem(...)
+ *   detectFramwork({ fs, frameworkList }) // returns the 'slug' field if detected, otherwise null
+ *
+ * @todo Will be used by the detect-eligible-projects API endpoint for a given git url.
+ */
+export const monorepoManagers: Array<Framework> = [
+  {
+    name: 'Turborepo',
+    slug: 'turbo',
+    detectors: {
+      every: [
+        {
+          path: 'turbo.json',
+        },
+      ],
+    },
+    // unused props - needed for typescript
```

Comment:
This seems like the wrong solution. Perhaps we need a new type

Reply:
I suspect we'll find an use for them in the future, but for now they're unused.  These changes were based on @Ethan-Arrowood's comments here https://github.com/vercel/vercel/pull/7737#discussion_r865340977

Reply:
The suggestion there mentions Omit

Let use something like this:

```
Array<Omit<Framework, 'logo' | 'settings' | 'getOutputDirName'>>
```

Reply:
<img width="865" alt="image" src="https://user-images.githubusercontent.com/1278714/169107337-0b9a893c-aef0-4bed-bae1-88ee999b325c.png">